### DISCUSSION TITLE
Support for ActiveSync Policy

### DISCUSSION THREAD

**Discussion by SinghNanak** [Z-Hub/Z-Push#79] [Q&A] [answered]
does Z-Push's implementation of ActiveSync support [Mobile device mailbox policy.
](https://learn.microsoft.com/en-us/exchange/clients/exchange-activesync/mobile-device-mailbox-policies?view=exchserver-2019)

**Comment by matidau** [comment, marked as answer]
Hi @nsdhanoa,

Have a look at src/policies.ini

And this is a copy of the z-hub wiki page
https://github.com/Z-Hub/Z-Push/wiki/Mobile-Policies

**Reply by SinghNanak** [reply]
thank you @matidau,
I see "global mobile policies on a server wide level." does that mean there could only be one policy that would be implemented on all mobile devices and email accounts.

**Reply by matidau** [reply]
Yes, if you want per device/user configuration please raise a feature/enhancement issue.
